components/Common/CodeBox/CodeTabs/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { ArrowUpRightIcon } from '@heroicons/react/24/solid';
+import type { FC, PropsWithChildren } from 'react';
+
+import type { TabsProps } from '@/components/Common/Tabs';
+import Tabs from '@/components/Common/Tabs';
+import LocalizedLink from '@/components/LocalizedLink';
+
+import styles from './index.module.css';
+
+type CodeTabsProps = Pick<
+  TabsProps,
+  'tabs' | 'onValueChange' | 'defaultValue'
+> & {
+  linkUrl?: string;
+  linkText?: string;
+};
+
+const CodeTabs: FC<PropsWithChildren<CodeTabsProps>> = props => {
```

Comment:
We recommend destructuring props, instead of assigning to a variable. That's what we've been doing so far.